tests/nightly/test_np_large_array.py

```diff
@@ -453,13 +456,14 @@ def batch_check(x1, x2, axises, shapes):
 # backward not working https://github.com/apache/incubator-mxnet/issues/18952
```

Comment:
outdated comment?

Reply:
Right backward actually works I was calling it wrong. removing the comment and issue sorry

Reply:
Right backward actually works I was calling it wrong. removing the comment and issue sorryMaybe I am doing this wrong